inf_post/include/infix.h

```diff
@@ -0,0 +1,4 @@
+#include <string>
+#include <stack>
```

Comment:
You have included <stack> but it is not used in this file. don't do so.

Reply:
+